src/service/subscriberLoader.ts

```diff
@@ -1,15 +1,9 @@
 import { resolve, join } from 'path';
-import fs = require('fs');
-import {
-  SubscriberV2,
-  Subscribers,
-  SubscriberTuple,
-  SubscriberV1,
-} from '../subscriber';
+import fs from 'fs';
+import { SubscriberV2, Subscribers, SubscriberTuple } from '../subscriber';
```

Comment:
Codacy has a fix for the issue: [Replace `V2,·Subscribers,·SubscriberTuple` with `s,·SubscriberTuple,·SubscriberV2`](https://app.codacy.com/gh/deliveryhero/hfc-pubsub/pullRequest?prid=8106467)
```suggestion
import { Subscribers, SubscriberTuple, SubscriberV2 } from '../subscriber';
```